app/controllers/users/sessions_controller.rb

```diff
@@ -44,22 +50,29 @@ def require_no_authentication
   end
 
   def user_params
-    params.require(:user).permit(:email, :password, :remember_me, :otp_attempt)
+    params.require(:user).permit(:email, :password, :remember_me, :otp_attempt, :mobile_number)
```

Comment:
Is `user_params` or `configure_sign_in_params` what we're using to add mobile_number to permitted params, or both?

My guess is we can delete line 14 and the configure_sign_in_params method entirely, seeing as if I comment it out, then `spec/features/staff/users_can_sign_in_spec` still passes 🤔

Reply:
Mm. I think you're right. I think this is still there in a cargo-culty sense; since we're rendering every phase of the process now and Devise isn't, I think it's a hangover from a time when we weren't and Devise needed To Be Told. Let's remove.